shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -163,6 +166,8 @@ - (instancetype)initWithViewController:(FlutterViewController*)viewController {
     _textInputContext = [[NSTextInputContext alloc] initWithClient:self];
     _previouslyPressedFlags = 0;
 
+    self.flutterViewController = viewController;
```

Comment:
nice

Reply:
I am not an expert on objective c, is this the comment practice to use setter instead of _flutterViewController  = viewController; ?

Reply:
`flutterViewController` is declared as property, not ivar. I'm not entirely sure why given that these properties are only used in `FlutterTextInputPlugin` implementation. That said, all other properties seem to be accessed though `self.` so I don't quite see why this one should be different.

Reply:
I found this style guide suggested to use ivar https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmPractical.html

Reply:
You're right, it's in constructor. I'll change it to ivar.

Reply:
[Fixed](https://github.com/flutter/engine/pull/25524/commits/ac9610727f08f2c9b3c72cc69940541af80e4638#diff-bf62e3c20fb34cec64858961d271a62ccbde5a2bd822f7e5747d7b5886b63ff2R169).